test/e2e/Makefile

```diff
@@ -0,0 +1,16 @@
+docker:
+	docker build --tag tendermint/e2e-node -f docker/Dockerfile ../..
+
+ci: runner
+	./build/runner -f networks/ci.toml
+
+app:
+	# We need to build support for database backends into the app in
+	# order to build a binary with a Tendermint node in it (for built-in
+	# ABCI testing).
+	go build -o build/app -tags badgerdb,boltdb,cleveldb,rocksdb ./app
+
+runner:
+	go build -o build/runner ./runner
+
+.PHONY: app ci docker runner
```

Comment:
should this make file be imported into the main makefile?

Reply:
I'd rather keep it separate, it's much easier to reason about and work with when it isn't tangled up with everything else.